### PR TITLE
Gross median weekly pay

### PR DESCRIPTION
**Pull request by santhosh-thangavel**
Made changes to area and period column mapping

#8

### CODE REVIEW

File: gross-median-weekly-pay/gross-median-weekly-pay.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
         "Period": {
             "type": "dimension",
             "from_existing": "http://purl.org/linked-data/sdmx/2009/dimension#refPeriod",
+            "label": "Year",
```
Comment (sazzadur-ons) [inline gross-median-weekly-pay/gross-median-weekly-pay.json:21]: "Period":{
"from_template": "year"
}
would be fine. It does not need to define further.

File: gross-median-weekly-pay/gross-median-weekly-pay.json [hunk 2]
```diff
@@ -18,6 +18,7 @@
...
             "cell_uri_template": "http://reference.data.gov.uk/id/{+period}"
         },
         "Area": {
```
Comment (sazzadur-ons) [inline gross-median-weekly-pay/gross-median-weekly-pay.json:24]: "Area": {
"from_template":""statistical-geography"
}
would be fine. It does not need to define further.

File: gross-median-weekly-pay/gross-median-weekly-pay.json (hunk 1 above)
Comment (santhosh-thangavel) [reply gross-median-weekly-pay/gross-median-weekly-pay.json]: Made changes as per the comment

File: gross-median-weekly-pay/gross-median-weekly-pay.json (hunk 2 above)
Comment (santhosh-thangavel) [reply gross-median-weekly-pay/gross-median-weekly-pay.json:21]: made changes as per the comment

File: gross-median-weekly-pay/gross-median-weekly-pay.json [hunk 3]
```diff
@@ -53,4 +59,4 @@
             "from_template": "observation-status"
         }
     }
-}
+}
```
Comment (sazzadur-ons) [inline gross-median-weekly-pay/gross-median-weekly-pay.json:56]: "Lower Confidence Interval (95%)":{
            "type": "attribute"
        },
        "Upper Confidence Interval (95%)":{
            "type": "attribute"
        }
both should be 'observations' type (values of these two columns in the csv file are observations).

Comment (santhosh-thangavel) [reply gross-median-weekly-pay/gross-median-weekly-pay.json:56]: Made changes

**Pull request merged**